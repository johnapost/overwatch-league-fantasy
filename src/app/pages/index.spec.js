import React from "react";
import { shallow } from "enzyme";
import Index from "./index";

describe("Index", () => {
  it("should render fine", () => {
    const wrapper = shallow(<Index />);
    expect(wrapper).toMatchSnapshot();
  });
});
