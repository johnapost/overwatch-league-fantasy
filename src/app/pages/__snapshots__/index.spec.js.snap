// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render fine 1`] = `
"<div class=\\"container\\"><main><div class=\\"ant-row\\"><div class=\\"ant-col-sm-6 ant-col-md-4\\"></div><div class=\\"ant-col-sm-12 ant-col-md-16\\"><div class=\\"jsx-2964576322 wrapper\\"><div class=\\"ant-card ant-card-bordered\\"><div class=\\"ant-card-body\\">This site is a work in progress and is not ready for usage at all! Any data is subject to deletion while work continues.</div></div></div></div></div><div class=\\"jsx-2964576322 table-wrapper\\"><div class=\\"ant-row table\\"><div class=\\"ant-col-md-4\\"></div><div class=\\"ant-col-md-16\\"></div></div></div></main></div><footer class=\\"jsx-1140642571\\"><div class=\\"ant-row\\"><div class=\\"ant-col-sm-6 ant-col-md-4\\"></div><div class=\\"ant-col-sm-12 ant-col-md-16\\"><p class=\\"jsx-1140642571\\">Special thanks to Overwatch and the Overwatch League for existing.</p><p class=\\"jsx-1140642571\\">All trademarks belong to their owners.</p></div></div></footer><style>
      .container {
        background-image: url(static/diamond_upholstery.png);
        background-attachment: fixed;
      }

      header {
        height: 59px;
      }

      main {
        min-height: 100vh;
        overflow-x: hidden;
      }

      @media (min-device-pixel-ratio: 2) {
        .container {
          background-image: url(static/diamond_upholstery_@2x.png);
        }
      }
    </style>"
`;
